build.sbt

```diff
@@ -496,13 +512,15 @@ lazy val root = project
   .settings(formattingSettings)
   .settings(disableDocs)
   .settings(disablePublishing)
+  .settings(crossScalaVersions := Seq(scala212))
```

Comment:
Why do we need this here?

Reply:
I pulled it up from [disablePublishing](https://github.com/playframework/play-ws/pull/301/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceL212). `integration-tests` overrode it to include all 3 versions (the ThisBuild default), and `bench` needs to do the same things or has build failures from the dependencies. Values of the `crossScalaVersions ` settingKey should be equivalent to what was there before.

git blame points to https://github.com/playframework/play-ws/pull/244. No idea what's happening there, although I assume Eugene knows what he's doing.